adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDDFunctions.scala

```diff
@@ -21,18 +21,14 @@ import org.apache.spark.rdd.RDD
 import org.bdgenomics.adam.converters.AlignmentRecordConverter
 import org.bdgenomics.adam.models.SequenceRecord
 import org.bdgenomics.adam.rdd.ADAMSequenceDictionaryRDDAggregator
+import org.apache.spark.Logging
 import org.bdgenomics.formats.avro._
 import scala.collection.JavaConversions._
 
-class FragmentRDDFunctions(rdd: RDD[Fragment]) extends ADAMSequenceDictionaryRDDAggregator[Fragment](rdd) {
+class FragmentRDDFunctions(rdd: RDD[Fragment]) extends Serializable with Logging {
```

Comment:
Sounds good to me, I'm always in favor of removing unnecessary code.

I wonder if this changes anyone's opinion on this commit https://github.com/ryan-williams/adam/commit/c5a8f51f7d1bc38cc80f9f0699b5ac92947ecf31 that adds extension to some of the Functions classes.